test/s3path.jl

```diff
@@ -432,6 +432,17 @@ function s3path_tests(config)
         rm(json_path)
     end
 
+    @testset "Arrow <-> S3Path (de)serialization" begin
+        ver = String('A':'Z') * String('0':'5')
+        paths = [
+            S3Path("s3://$(bucket_name)/a"), S3Path("s3://$(bucket_name)/b?versionId=$ver")
+        ]
```

Comment:
More inline with BlueStyle
```suggestion
        paths = [
            S3Path("s3://$(bucket_name)/a"),
            S3Path("s3://$(bucket_name)/b?versionId=$ver"),
        ]
```